ta/ta.mk

```diff
@@ -22,8 +22,8 @@ $(sm)-platform-cppflags += -DMBEDTLS_SELF_TEST
 endif
 
 ifeq ($(CFG_TEE_TA_MALLOC_DEBUG),y)
-# Build malloc debug code into libutils: (mdbg_malloc(), mdbg_free(),
-# mdbg_check(), etc.).
+# Build malloc debug code into libutils: (__mdbg_alloc(), mdbg_free(),
+# and mdbg_check().).
```

Comment:
It looks kind of weird to have `__mdbg_alloc()` and no `mdbg_malloc()` although there is `mdbg_free()`. I think it would be better to not mention these implementation details. TAs are not supposed to depend on them. Better mention what it does (more checks, stats?)

Reply:
OK, I'll update.